sympy/physics/vector/vector.py

```diff
@@ -751,20 +751,57 @@ def angle_between(self, vec):
         return angle
 
     def free_symbols(self, reference_frame):
-        """
-        Returns the free symbols in the measure numbers of the vector
+        """Returns the free symbols in the measure numbers of the vector
         expressed in the given reference frame.
 
-        Parameter
-        =========
+        Parameters
+        ==========
+        reference_frame : ReferenceFrame
+            The frame with respect to which the free symbols of the given
+            vector is to be determined.
+
+        Returns
+        =======
+        set of Symbol
+            set of symbols present in the measure numbers of
+            ``reference_frame``.
+
+        See Also
+        ========
+
+        - :meth:`~sympy.core.basic.Basic.free_symbols`
```

Comment:
I'm struggling to get these cross references working. I get this error:
```
/home/moorepants/src/sympy/sympy/physics/vector/vector.py:docstring of sympy.physics.vector.vector.Vector.free_dynamicsymbols:24: WARNING: py:obj reference target not found: -
/home/moorepants/src/sympy/sympy/physics/vector/vector.py:docstring of sympy.physics.vector.vector.Vector.free_symbols:24: WARNING: py:obj reference target not found: -
```
I've also tried `:obj:`~.free_symbols`` and `:obj:`sympy.physics.mechanics.functions.find_dynamicsymbols` as suggested in the docstring style guide and get the same error.

Reply:
For a method I think it should be `:meth:~.Basic.free_symbols` but for a property method it's probably better to use `:attr:` rather than `:meth:`.

Reply:
I tried:

```rst
        - :attr:`~.Basic.free_symbols`
        - :func:`~.find_dynamicsymbols`
```

And I get the same errors.

Reply:
Maybe it's not possible. I missed the fact that you're adding this in the "see also" section which is a numpydoc thing that seems to have a completely different set of referencing rules. I can't find how to refer to a method/attribute in the see also section in the docs for numpy doc.

Reply:
I wouldn't think that would matter. The fully qualified name should work in all cases, I would have thought. I just deleted them.

Reply:
I think that the "see also" uses a completely different "namespace" from ordinary sphinx referencing so it does matter.

Reply:
Interesting.

Reply:
I see you figured this out, but yeah, the numpydoc extension takes the lines in "See Also" and rewrites them into cross references. It really ought to detect when the line looks like a cross reference itself and just leave it alone (or at least give a better error message).

Reply:
Ah! I actually didn't figure it out. I just deleted it.